src/surge-xt/SurgeSynthEditor.cpp

```diff
@@ -330,7 +331,7 @@ void SurgeSynthEditor::resized()
     if (zfn != 1.0 && rezoomGuard == 0)
     {
         auto br = BlockRezoom(this);
-        adapter->setZoomFactor(round(adapter->getZoomFactor() * zfn), false);
+        adapter->setZoomFactor(round(adapter->getZoomFactor() * zfn), true);
```

Comment:
Uhhh why this change?

Reply:
My mistake, I was trying stuff out and forgot to revert it. Apols!